config/science_board_config.yaml

```diff
@@ -1,21 +1,26 @@
-led_map:
-  red: 0
-  blue: 1
-  green: 2
-  off: 3
-mosfet_device_map:
-  # These devices are assigned
-  # to as specific MOSFET device
-  arm_laser: 1
-  heater_0: 7
-  heater_2: 9
-  uv_led_carousel: 4
-  uv_led_end_effector: 1
-  white_led: 5
-serial:
-  baudrate: 38400
-  timeout: 0
-info:
-  max_error_count: 20
-  sleep: 0.01
-  uart_transmit_msg_len: 30
+led_map:
+  "red": 0
```

Comment:
i seriously don't understand why this caused issues smmhhh

Reply:
Cause the keyword had to be a string lol

Reply:
That's what we call CRINGE in the industry